### ISSUE TITLE
Confused about the Equation 6. in paper

### ISSUE THREAD

**Issue by Jordan-5i** [yahoo/object_relation_transformer#5]
hey, i am confused about the x_m-x_n and y_m-y_n when they are equal to zero under special condition using faster rcnn object detector, because center coordinates may be same and results to log(0).  How do solve this problem?

**Comment by simaoh** [comment]
@Jordan-5i You are absolutely right, about the corner case of two bounding boxes with the same center coordinates.

To cover that explicitly in equation 6,  one should replace |x_m-x_n| by min(|x_m-x_n|, eps) for some small constant eps>0.
In our code we do that with eps=0.001*w, and analogous computation for the y coordinates.

Here is the part of the code where we address this with `torch.clamp` (and also provide for the degenerate case of w=0, or h=0).
https://github.com/yahoo/object_relation_transformer/blob/f21674d5c1095fc104ff3b69bfa41cfeea7568db/misc/utils.py#L245-L255
